src/DetailsView/components/iframe-skipped-warning.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+
+export const IframeSkippedWarningContainerAutomationId = 'iframe-skipped-warning-container';
+
+export const IframeSkippedWarning = NamedFC('IframeSkippedWarning', () => (
+    <div data-automation-id={IframeSkippedWarningContainerAutomationId}>
+        There are iframes in the target page that are non-responsive. These frames have been skipped
```

Comment:
This is a good starting point, but I'm not sure this is actionable enough. What do we want a user who sees this warning to do about it, exactly? Perhaps we should include something like "If you see this warning consistently for a page where it isn't expected, please [file an issue](https://github.com/microsoft/accessibility-insights-web/issues/new)."? @ferBonnin , what are your thoughts?

Reply:
Or perhaps it would be more manageable to create a single known tracking issue and link to it, asking folks to comment on it with the site in question.

Reply:
Either of those options sound reasonable to me! I'll defer to @ferBonnin's judgement for specific wording/ how to manage issues.

Reply:
let's create a tracking issue and let users comment on it, the barrier is lower for the user.
What are the cases where the user would get this message unexpectedly? if there aren't iframes?

for the message, how about:
_If you get this warning in a page <reason to get it unexpectedly>, please [let us know](link to the issue)._

Reply:
> What are the cases where the user would get this message unexpectedly? if there aren't iframes?

Users shouldn't get this message when there aren't iframes on the page, but they might get it on pages where they expect the iframes to be responsive but they're not, for example, if the iframes are responding but very slowly. 

> for the message, how about: `If you get this warning in a page , please [let us know](link to the issue).`

Sounds good to me! I assume that would be in addition to the existing text?

Reply:
not sure I follow completely: in which cases would we want the user to let us know / file a bug?

> Sounds good to me! I assume that would be in addition to the existing text?

edited my message because I was using `<>` and the text between them didn't show 🙂 
Yes, that would be in addition to the text " There are iframes in the target page ..."

Reply:
Not sure if Dan has other thoughts, but IMO I think it would be interesting to get some example target pages where a user thinks that the iframes should be responsive but for some reason we aren't able to communicate with them (and as a result show this error). So it would be valuable for a user to file an issue if they are seeing this warning unexpectedly and think that the iframes should be/ are responsive. 

In this case, should we be a little more specific in the wording and include something like `If you get this warning unexpectedly in a page where you expect the iframes to be responsive, please [let us know](link to the issue).`

Reply:
I created a tracking issue to link to here: https://github.com/microsoft/accessibility-insights-web/issues/6347, please feel free to edit it directly if I missed anything!

Reply:
That message looks great to me!

Reply:
LGTM! I think the most common case will actually be that the person running the tool won't really know a lot of details about the iframes causing problems in the page, but I think that message should cover that case okay. Thanks for the update!